.github/actions/setup-action/action.yml

```diff
@@ -36,16 +36,20 @@ runs:
       uses: actions/checkout@v3
       if: |
         github.event_name == 'push' ||
-        github.event_name == 'pull_request_target' ||
         github.event_name == 'schedule' ||
         github.event_name == 'workflow_dispatch'
       with:
         ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Hm, for push/schedule/workflow_dispatch, we probably don't even need this checkout, do we?

Reply:
Using the pull_request event doesn't really make sense and we've already done the checkout previously to get access to this action

Reply:
yeah that's right. Changed and tested on https://github.com/Abacn/beam/actions/runs/5787197443/job/15683590845